chill-scala/src/main/scala/com/twitter/chill/KryoBase.scala

```diff
@@ -41,6 +43,14 @@ class KryoBase extends Kryo {
   def isFn(klass: Class[_]): Boolean =
     functions.find { _.isAssignableFrom(klass) }.isDefined
 
+  def isJavaLambda(klass: Class[_]): Boolean =
+    Java8ClosureRegistrar.areOnJava8 && klass.getName().indexOf('/') >= 0
+
+  override def getRegistration(klass: Class[_]) =
+    if (isJavaLambda(klass)) {
+      getClassResolver.getRegistration(classOf[ClosureSerializer.Closure])
```

Comment:
hmm, odd that this doesn't have coverage even though without it the tests failed.